pkg/resizer/csi_resizer.go

```diff
@@ -33,12 +33,19 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	storagev1listers "k8s.io/client-go/listers/storage/v1"
+
+	csitranslationlib "k8s.io/csi-translation-lib"
 	"k8s.io/klog"
 )
 
 const (
 	resizerSecretNameKey      = "csi.storage.k8s.io/resizer-secret-name"
 	resizerSecretNamespaceKey = "csi.storage.k8s.io/resizer-secret-namespace"
+
+	// VolumeResizerKey is key that will be used to store resizer used
```

Comment:
by "store resizer used" do you mean the value associated with the key will be the *resizer name* or some other field that represents the resizer. Could you clarify the comment a bit.

Reply:
also would be useful to note what populates this annotation and whether we can always expect it to be there or what to do if it doesn't exist

Reply:
FWIW it is populated by in-tree volume resizer and if in-tree migration is enabled for the plugin, the value corresponding to this key should be csi driver name (even though PV spec is in-tree).

Reply:
Also if migration is not enabled, the value of this key should be in-tree k8s driver.

Reply:
udpated